src/js/controller.js

```diff
@@ -793,16 +793,70 @@ export const MediaUIRequestHandlers = {
+        preferredLanguage
+      );
+    }
     updateTracksModeTo(TextTrackModes.SHOWING, tracks, tracksToUpdate);
   },
-  MEDIA_DISABLE_SUBTITLES_REQUEST: (media, event, controller) => {
+  MEDIA_DISABLE_SUBTITLES_REQUEST: (_media, event, controller) => {
     const tracks = getSubtitleTracks(controller);
     const { detail: tracksToUpdate = [] } = event;
     updateTracksModeTo(TextTrackModes.DISABLED, tracks, tracksToUpdate);
   },
+  MEDIA_TOGGLE_SUBTITLES_REQUEST: (_media, event, controller) => {
+    // NOTE: Like Element::toggleAttribute(), this event uses the detail for an optional "force"
+    // value. When present, this means "toggle to" "on" (aka showing, even if something's already showing)
+    // or "off" (aka disabled, even if all tracks are currently disabled).
+    // See, e.g.: https://developer.mozilla.org/en-US/docs/Web/API/Element/toggleAttribute#force (CJP)
+    const { detail: force } = event ?? {};
+    const tracks = getSubtitleTracks(controller);
+    const showingSubitleTracks = getShowingSubtitleTracks(controller);
+    const subtitlesShowing = !!showingSubitleTracks.length;
+    // If there are no tracks, this request doesn't matter, so we're done.
+    // If we already have showing subtitles and we want to force toggle "on", there's nothing left to do.
+    // If there are no showing subtitles and we want to force toggle "off", we're already done.
+    if (!tracks.length || (subtitlesShowing && force) || (!subtitlesShowing && force === false)) return;
+
+    if (subtitlesShowing) {
+      updateTracksModeTo(TextTrackModes.DISABLED, tracks, showingSubitleTracks);
+    } else {
+      let subTrack = tracks[0];
+      if (!controller?.hasAttribute('nosubtitleslangpref')) {
+        const subtitlesPref = globalThis.localStorage.getItem('media-chrome-pref-subtitles-lang');
+
+        const userLangPrefs = subtitlesPref
+          ? [subtitlesPref, ...globalThis.navigator.languages]
+          : globalThis.navigator.languages;
+        const preferredAvailableSubs = tracks.filter(textTrack => {
+          return userLangPrefs.some(lang => textTrack.language.toLowerCase().startsWith(lang.split('-')[0]));
+        }).sort((textTrackA, textTrackB) => {
+          const idxA = userLangPrefs.findIndex(lang => textTrackA.language.toLowerCase().startsWith(lang.split('-')[0]));
+          const idxB = userLangPrefs.findIndex(lang => textTrackB.language.toLowerCase().startsWith(lang.split('-')[0]));
+          return idxA - idxB;
+        });
+
+        // Since there may not have been any user preferred subs/cc match, keep the default (picking the first) as
+        // the subtitle track to show for these cases.
+        if (preferredAvailableSubs[0]) {
+          subTrack = preferredAvailableSubs[0];
+        }
+      }
+      const { language, label, kind } = subTrack;
+      updateTracksModeTo(
+        TextTrackModes.SHOWING,
+        tracks,
+        [{ language, label, kind }]
+      );
+    }
```

Comment:
I would suspect the `force` var still to be used somewhere here.
thinking through this...

Reply:
should be accounted for on https://github.com/muxinc/media-chrome/pull/770/files#diff-f47c60f82e0b188cc72db120e2829c4473753d20186183b4c2e1492169d61321R826 right?

Reply:
oh no you're right. We don't currently have a use case in our codebase but we should account for that. Updating.

Reply:
also just re-thought through this and my former self did account for that... def required some reasoning. The tl;dr:

1. The early bail cases for `forced` say "if we're already in the forced state we want, we don't have to do anything"
2. Then, we're only in cases where we used `forced` to cause the same state change we already would have without specifying `forced`.
3. So, whether we stipulated `forced` or not, we should be disabling subtitle tracks that are currently showing or showing a subtitle track because none are showing.